scripts/train.ipynb

```diff
@@ -165,11 +165,11 @@
     "import networkx as nx\n",
     "from datetime import datetime\n",
     "import collections\n",
-    "from itertools import chain\n",
```

Comment:
Is there any particular reason for rolling-back the train script as it's thought for a representative training ?